Makefile

```diff
@@ -56,7 +56,7 @@ VARIANT_VALUES = cardano_ada
 
 DEFINES += RESET_ON_CRASH
 # Enabling DEBUG flag will enable PRINTF and disable optimizations
-# DEVEL = 1
+DEVEL = 1
```

Comment:
What does this flag do ? If it's a debug option why put it in prod?

Reply:
I need to comment it, I activated it for local debug. Thx for noticing.